apps/api/src/database/database.naming-strategy.ts

```diff
@@ -0,0 +1,47 @@
+import { NamingStrategyInterface, DefaultNamingStrategy } from 'typeorm';
+import { snakeCase } from 'typeorm/util/StringUtils';
+
+export class DatabaseNamingStrategy
```

Comment:
cool I've never seen this, do the decisions here need to be documented?

Reply:
This is part of the FW NestJS boilerplate code. I don't think it's a significant decision/worth documenting.